src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -204,22 +204,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     packages references.
     ============================================================
     -->
-  <UsingTask TaskName="Microsoft.NET.Build.Tasks.CollectResolvedSDKReferencesDesignTime"
-             AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
-  
-  <Target Name="CollectSDKReferencesDesignTime"
-          Returns="@(_SDKReference)"
-          DependsOnTargets="ResolvePackageDependenciesDesignTime">
+  <UsingTask TaskName="Microsoft.NET.Build.Tasks.CollectSDKReferencesDesignTime"
+         AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
 
-    <CollectResolvedSDKReferencesDesignTime
-          ResolvedSdkReferences="@(SdkReference)"
-          DependenciesDesignTime="@(_DependenciesDesignTime)">
+  <Target Name="CollectSDKReferencesDesignTime"
+          Returns="@(_SDKReference)">
```

Comment:
Ah, i see, so there maybe more PackageReferences coming from CollectPackageReferences . Sounds good i will depend on it here.